src/LanguageServer/Impl/Formatting/LineFormatter.cs

```diff
@@ -197,6 +197,9 @@ public class LineFormatter {
                     case TokenKind.BitwiseAndEqual:
                     case TokenKind.BitwiseOrEqual:
                     case TokenKind.ExclusiveOrEqual:
+                    // Used in expressions as an operator
```

Comment:
Drop the comment; it doesn't add anything (because all of these are operators used in expressions...)